manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml

```diff
@@ -73,6 +73,9 @@ rules:
 #- apiGroups: [ "networktopology.diktyo.x-k8s.io" ]
 #  resources: [ "networktopologies" ]
 #  verbs: [ "get", "list", "watch", "create", "delete", "update", "patch" ]
+- apiGroups: ["security-profiles-operator.x-k8s.io"]
+  resources: ["seccompprofiles", "profilebindings"]
+  verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
```

Comment:
Could you comment it out? for newly introduced plugin, we don't enable it until it's tested enough.